src/charts/stacked-area.js

```diff
@@ -313,28 +315,45 @@ define(function(require){
         function buildAxis() {
             let minor, major;
 
-            if (xAxisFormat === 'custom' && typeof xAxisCustomFormat === 'string') {
-                minor = {
-                    tick: xTicks,
-                    format:  d3TimeFormat.timeFormat(xAxisCustomFormat)
-                };
+            if(xAxisValueType === 'number') {
+                minor = getSortedNumberAxis(dataSorted, width);
                 major = null;
+
+                if(xAxisScale === 'logarithmic') {
+                    xAxis = d3Axis.axisBottom(xScale)
+                        .ticks(minor.tick, "e")
+                        .tickFormat(function (d) {
+                            var log = Math.log(d) / Math.LN10;
```

Comment:
Could you use const here?

Reply:
Oh, yes sure - did not see that. Will update it.